lib/utils/getStakingRewards/index.ts

```diff
@@ -9,7 +9,7 @@ export const getStakingRewards = async (
   days: number,
   blockRate: number
 ): Promise<number> => {
-  const provider = getJsonRpcProvider();
+  const provider = getJsonRpcProvider(urls.infuraRpc);
```

Comment:
once you implement my comment above, you will need to be passing down `INFURA_ID` rather than the entire url.

```ts
const provider = getJsonRpcProvider({ infuraId: INFURA_ID });
```